static_test_driver/autosequence.ino

```diff
@@ -146,6 +151,17 @@ void run_control() {
         Serial.println(F("Thrust below critical level"));
         abort_autosequence();
       }
+       // Check that the pressure of the fuel and oxidizer line have not exceeded the maximum limit
```

Comment:
Are you sure that we only want this check after ignition and mainstage?  Seems like we might want to abort before ignition if the line pressures are too high.  Unless those sensors are downstream of the valves, in which case ignore this comment.

Reply:
Good catch. I'll double check that with Jame as I'm not positive

Reply:
We do want to check pressures after the prestage valves have opened before ignition.